docs/entities/livingentity.md

```diff
@@ -0,0 +1,278 @@
+
+### Spawn Eggs
+
+It is common (though not required) to [register] a spawn egg for mobs. This is done through the `SpawnEggItem` class, which has been patched by NeoForge to do some extra setup, such as registering the color handler and adding the spawn egg to the internal `SpawnEggItem` -> `EntityType` map.
+
+```java
+// Assume we have a DeferredRegister.Items called ITEMS
+DeferredItem<SpawnEggItem> MY_ENTITY_SPAWN_EGG = ITEMS.registerItem("my_entity_spawn_egg",
+    properties -> new SpawnEggItem(
+        // The entity type to spawn.
+        MY_ENTITY_TYPE.get(),
+        // The colors to use for the spawn egg. The first one is the base/background color,
+        // the second one is the spots/highlight color.
+        0xff0000,
+        0x00ff00,
+        // The properties passed into the lambda, with any additional setup.
+        properties
+    ));
+```
+
+As an item like any other, the item should be added to a [creative tab][creative], and a [model] and [translation] should be added.
+
+### Natural Spawning
+
+_See also [Entities/`MobCategory`][mobcategory], [Worldgen/Biome Modifers/Add Spawns][addspawns], [Worldgen/Biome Modifers/Add Spawn Costs][addspawncosts]; and [Spawn Cycle][spawncycle] on the [Minecraft Wiki][mcwiki]._
+
+Natural spawning is performed every tick for entities where `MobCategory#isFriendly()` is true, and every 400 ticks (\= 20 seconds) for entities where `MobCategory#isFriendly()` is false. If `MobCategory#isPersistent()` returns true, this process additionally also happens on chunk generation.
+
+For each chunk and mob category, it is checked whether there are less than `MobCategory#getMaxInstancesPerChunk() * loadedChunks / 289` in the world. Additionally, for each chunk, it is required that there are less than `MobCategory#getMaxInstancesPerChunk()` entities of that `MobCategory` near at least one player (near means that the distance between mob and player \<\= 128) for spawning of that `MobCategory` to occur.
+
+If the conditions are met, an entry is randomly chosen from the relevant biome's spawn data, and spawning occurs if a suitable position can be found. There are at most three attempts to find a random position; if no position can be found, no spawning will occur.
+
+## AI and Navigation
+
+One of the `Mob` subclass's main features is the AI system, as this is of course not applicable to players or armor stands. It tells the mob where to go, what to do and whom to attack.
+
+There are two implementations of this system: the goal system and the brain system. Both of these systems then use a subclass of `PathNavigation` to calculate the path to their desired position.
+
+### Goals
+
+:::info
+This section is a work in progress.
+:::
+
+### Brains
+
+:::info
+This section is a work in progress.
+:::
+
+### Navigation
+
+:::info
+This section is a work in progress.
+:::
```

Comment:
Work in progress?

Reply:
Yes, and these sections will not be covered by the PR due to out-of-scope (and also because I do not have the energy to read into these systems at present).

Reply:
I would just remove the sections for now and stick them in an issue if they're not already.